components/PresaleHeader.tsx

```diff
@@ -4,54 +4,80 @@ import { calcRem } from '../utils/styles'
+import GlobalVars from './GlobalVars'
+import { Countdown } from './countdown/Countdown'
+import moment from 'moment'
 
 export function PresaleHeader({ className }: SectionProps) {
-  // TODO remove Hidden when implementing responsive
+
   return (
-    <div
+      <div
       className={classNames(
-        'hidden desktop:flex w-full absolute bg-blue text-white font-graphik justify-between h-28',
+        'grid grid-cols-3 items-center justify-items-start bg-blue text-white gap-x-2 font-graphik laptop:h-28 ',
+        'grid-cols-teasingPresaleHeader laptop:grid-cols-presaleHeader w-full',
         className
-      )}
-    >
-      <div
-        className="self-center ml-16"
-        style={{ fontSize: calcRem(18), width: calcRem(334) }}
+      )} 
+      style={{
+        height: calcRem(80)
+      }}
       >
-        PFX presale starts on 43rd July 2021
-      </div>
-      <div className="flex">
+        <div 
+          className={classNames("laptop:hidden bg-blue-light grid items-center h-full w-full")}
+          style={{
+            fontSize: calcRem(10)
+          }}
+        >
+          <div className="-rotate-90 text-center">
+            { GlobalVars.showPresaleStartDate ? "NOW" : "SOON" }
+          </div>
+        </div>
         <div
-          className="container flex justify-between items-center space-x-5 mx-24"
-          style={{ width: calcRem(312) }}
+          className={classNames(GlobalVars.showPresaleStartDate ? "laptop:self-center laptop:ml-16 pl-2" : 'hidden')}
+          style={{ fontSize: calcRem(18) }}
         >
-          <PresaleTimerBox>10d</PresaleTimerBox>
-          <PresaleTimerBox>23h</PresaleTimerBox>
-          <PresaleTimerBox>15m</PresaleTimerBox>
+          { GlobalVars.presaleStartUTCDate.isAfter(moment.utc()) && 
+            'PFX presale starts on ' + GlobalVars.presaleStartUTCDate.format("MMMM Do YYYY, h:mm:ss A z")
+          }
+          { GlobalVars.presaleStartUTCDate.isBefore(moment.utc()) && 
+            'PFX presale ends on ' + GlobalVars.presaleEndUTCDate.format("dddd, MMMM Do YYYY, h:mm:ss A z")
+          }
         </div>
-        <div className="mx-12 self-center">
-          <Link href="/presale">
-            <a>
-              <RightArrow className="fill-current" />
-            </a>
-          </Link>
+        <div className={classNames(GlobalVars.showPresaleStartDate ? 'hidden' : 'ml-1 laptop:justify-center laptop:ml-8')}>
+          PFX presale starts mid-September
+        </div>
+        <a 
+          className={classNames(GlobalVars.showPresaleStartDate ? "laptop:hidden" : "", 
+            "mr-4 bg-white text-blue grid items-center justify-self-end text-center rounded-full px-2 font-semibold ")
+          } 
+          style={{
+            height: calcRem(50),
+            width: calcRem(140),
+            fontSize: calcRem(14)
+          }}
+          href={GlobalVars.showPresaleStartDate ? "/presale" : "https://t.me/pfxannouncements"}
```

Comment:
When we have "MORE INFO" displayed, the link should be `_blank`.

When making a link `_blank`, make sure you use the same structure as our other `_blank` links (using `noreferrer`, etc).